### PR TITLE
Optimize #602 remove b clone2 source

### PR DESCRIPTION
**Pull request by jan-petr**
### Linked issue

#602 

### How to test

Test by running flavors - enough to run DCM2BIDS - further testing not needed and might work only after merging all the other current branches that all work with the import.

### CODE REVIEW

**Review by MichaelStritt — APPROVED** [review]
Seems fine to me 👍

**Comment by MichaelStritt** [comment]
### Tests

I tested using two GE flavors, everything seems fine in the comparison step of the BIDS full pipeline:

```
==============================================================================================
 ________                      __                                 ______    ______   __        
/        |                    /  |                               /      \  /      \ /  |      
########/  __    __   ______  ## |  ______    ______    ______  /######  |/######  |## |      
## |__    /  \  /  | /      \ ## | /      \  /      \  /      \ ## |__## |## \__##/ ## |      
##    |   ##  \/##/ /######  |## |/######  |/######  |/######  |##    ## |##      \ ## |      
#####/     ##  ##<  ## |  ## |## |## |  ## |## |  ##/ ##    ## |######## | ######  |## |      
## |_____  /####  \ ## |__## |## |## \__## |## |      ########/ ## |  ## |/  \__## |## |_____ 
##       |/##/ ##  |##    ##/ ## |##    ##/ ## |      ##       |## |  ## |##    ##/ ##       |
########/ ##/   ##/ #######/  ##/  ######/  ##/        #######/ ##/   ##/  ######/  ########/ 
                    ## |                                                                      
                    ## |                                                                      
                    ##/  

==================================== ExploreASL Settings =====================================
Dataset Root        
Import Modules      
Process Modules     
bPause              False
iWorker             1
nWorkers            1
==============================================================================================
ExploreASL v1.7.0_BETA initialized ... 
Dataset: GE_PCASL_3Dspiral_Product_pharma
Dataset: GE_PCASL_3Dspiral_volunteer
```

File: Functions/xASL_init_printSettings.m [hunk 1]
```diff
@@ -23,7 +23,7 @@ function xASL_init_printSettings(x)
     %% Fallbacks
```
Comment (HenkMutsaerts) [inline Functions/xASL_init_printSettings.m:23]: Are these fallbacks the printed lines? Then I would make this clear in the variable name.
Also, the printed lines can be made a bit more reader friendly. `NII` = `NIfTI`, and `NIfTI` is actually `legacy` right? Why is `DEFACE` big caps and `Structural` not? The `NIfTI` phase should be fased out soon anyway, so I wouldn't care too much about that naming :)

File: Modules/SubModule_Import/xASL_imp_Deface.m [hunk 2]
```diff
@@ -17,22 +17,20 @@ function xASL_imp_Anonymize(imPar)
...
-% EXAMPLE:     xASL_imp_Anonymize(imPar);
+% EXAMPLE:     xASL_imp_Deface(imPar);
 % __________________________________
 % Copyright 2015-2021 ExploreASL
 
-    %% Anonymize
-    
-    % 1. Iterate over list of subjects
+    %% 1. Iterate over list of subjects
     listSubjects = xASL_adm_GetFileList(imPar.TempRoot,[],false,[],true);
```
Comment (HenkMutsaerts) [inline Modules/SubModule_Import/xASL_imp_Deface.m:25]: Not important for now as I guess nobody uses this function (yet), but this doesn't give any result, it looks for folders without a name in the `TempRoot` folder :) Or does the second input argument default to something (it shouldn't I believe).

**Review by HenkMutsaerts — CHANGES_REQUESTED** [review]
Few minor issues but looks good overall!

File: Modules/SubModule_Import/xASL_imp_Deface.m (hunk 2 above)
Comment (MichaelStritt) [reply Modules/SubModule_Import/xASL_imp_Deface.m]: There really seems to be a bug related to all the renaming and stuff. Another issue here is that we have no nice way to test if the defacing happened or not, yet.

Comment (MichaelStritt) [reply Modules/SubModule_Import/xASL_imp_Deface.m]: Fix of the first problem: 490c59c

File: Functions/xASL_init_printSettings.m (hunk 1 above)
Comment (jan-petr) [reply Functions/xASL_init_printSettings.m:23]: You mean NII in DCM2NII - this is not legacy, this is the middle step of DCM->NII->BIDS. Although the Legacy format is used there, we still need this step. In the future, we will probably try to do this smarter, but we will step have it.

My proposed solution:
- Put also `STRUCTURAL` and `POPULATION` in capitals.
- Keep NII.

Agreed?

File: Modules/SubModule_Import/xASL_imp_Deface.m (hunk 2 above)
Comment (MichaelStritt) [reply Modules/SubModule_Import/xASL_imp_Deface.m]: Fix of the bug: e94cab3

Comment (jan-petr) [reply Modules/SubModule_Import/xASL_imp_Deface.m]: Weird. It should of course work. But maybe something happened with the directories during all the revamping. This is not a function we were testing. I will fix this.

Comment (jan-petr) [reply Modules/SubModule_Import/xASL_imp_Deface.m]: Seems that Michael already did that :D

Comment (jan-petr) [reply Modules/SubModule_Import/xASL_imp_Deface.m]: Tested and all works. Good work Michael.

File: Functions/xASL_init_printSettings.m (hunk 1 above)
Comment (jan-petr) [reply Functions/xASL_init_printSettings.m:23]: Done - do you agree?

Comment (MichaelStritt) [reply Functions/xASL_init_printSettings.m]: ### Test

```
==============================================================================================
 ________                      __                                 ______    ______   __        
/        |                    /  |                               /      \  /      \ /  |      
########/  __    __   ______  ## |  ______    ______    ______  /######  |/######  |## |      
## |__    /  \  /  | /      \ ## | /      \  /      \  /      \ ## |__## |## \__##/ ## |      
##    |   ##  \/##/ /######  |## |/######  |/######  |/######  |##    ## |##      \ ## |      
#####/     ##  ##<  ## |  ## |## |## |  ## |## |  ##/ ##    ## |######## | ######  |## |      
## |_____  /####  \ ## |__## |## |## \__## |## |      ########/ ## |  ## |/  \__## |## |_____ 
##       |/##/ ##  |##    ##/ ## |##    ##/ ## |      ##       |## |  ## |##    ##/ ##       |
########/ ##/   ##/ #######/  ##/  ######/  ##/        #######/ ##/   ##/  ######/  ########/ 
                    ## |                                                                      
                    ## |                                                                      
                    ##/  

==================================== ExploreASL Settings =====================================
Dataset Root        ...ATLAB\m.stritt\Server_xASL\New_TestData\Philips_PCASL_3DGRASE_volunteer
Import Modules      DCM2NII NII2BIDS DEFACE BIDS2LEGACY
Process Modules     STRUCTURAL ASL POPULATION 
bPause              False
iWorker             1
nWorkers            1
==============================================================================================
```

👍

**Review by HenkMutsaerts — CHANGES_REQUESTED** [review]
Cannot find the `fallback` variable anymore, but where I see the word `fallback` this should be `default` instead right? Just to make it understandable. Comments should explain what is happening, not only from a debugger's perspective :)

So let's just pay a bit more attention to commenting, otherwise this thing is really going to bite our back soon. Otherwise it looks good!

**Comment by jan-petr** [comment]
> Cannot find the `fallback` variable anymore, but where I see the word `fallback` this should be `default` instead right? Just to make it understandable. Comments should explain what is happening, not only from a debugger's perspective :)
> 
> So let's just pay a bit more attention to commenting, otherwise this thing is really going to bite our back soon. Otherwise it looks good!

I have removed `fallbacks` completely :) Because the default value was "", so I just reorganized the script a bit that we don't even need those.

**Comment by HenkMutsaerts** [comment]
> > Cannot find the `fallback` variable anymore, but where I see the word `fallback` this should be `default` instead right? Just to make it understandable. Comments should explain what is happening, not only from a debugger's perspective :)
> > So let's just pay a bit more attention to commenting, otherwise this thing is really going to bite our back soon. Otherwise it looks good!
> 
> I have removed `fallbacks` completely :) Because the default value was "", so I just reorganized the script a bit that we don't even need those.

It said "fallback" somewhere in the comment ;) If you change that to "default" then I am happy right?

**Comment by jan-petr** [comment]
> It said "fallback" somewhere in the comment ;) If you change that to "default" then I am happy right?

Yes exactly. Actually the 'fallback'/'default' setting is now completely removed. As we directly print stuff, without even setting the variable to something - so that comment was implemented and no more relevant.

Read to approve?!

**Review by HenkMutsaerts — APPROVED** [review]

**Pull request merged**